packages/global.d.ts

```diff
@@ -9,6 +9,7 @@ declare var __ESM_BUNDLER__: boolean
 declare var __ESM_BROWSER__: boolean
 declare var __CJS__: boolean
 declare var __SSR__: boolean
+declare var __VUE_SSR_SETTERS__: Array<(v: boolean) => void>
```

Comment:
This was added to address TypeScript type check errors in the test file.

Reply:
Let me know if it's undesirable to declare `__VUE_SSR_SETTERS__`. Maybe that exposes this type and it's not intended to be public. If so, is there a better way to type that interface in the unit test?

Reply:
Maybe we can export `setInSSRSetupState` and import it into `apiWatch.spec.ts` using a relative path.
https://github.com/vuejs/core/blob/40193696b3564202173ac0367e4b3ae48c4ffb84/packages/runtime-core/src/component.ts#L715